src/scene/vcFlythrough.cpp

```diff
@@ -113,6 +113,8 @@ void vcFlythrough::AddToScene(vcState *pProgramState, vcRenderData *pRenderData)
 
       for (size_t i = offset + 1; i < m_flightPoints.length; ++i)
         m_flightPoints[i].time += pProgramState->deltaTime;
+
+      m_timeLength += pProgramState->deltaTime;
```

Comment:
Probably best to set this to `m_flightPoints[m_flightPoints.length - 1].time` instead of relying on the FPU to produce the same result.